views/Geographic/GeographicMap.svelte

```diff
@@ -40,5 +68,25 @@
         {/each}
       </g>
     {/if}
+    <g class="q-choropleth-annotations">
+      {#each annotations as { id, coordinates }}
+        <AnnotationPointWithLine
+          id = {id}
+          radius = {annotationRadius}
+          coordinates = {coordinates} />
+      {/each}
+      <g class="q-choropleth-features">
+        <!--
+          We're finally adding the features with annotations here and not before, because then we can make sure
+          that those features are "on top" of everything else and the border around them is drawn correctly.
+        -->
```

Comment:
Not really sure what you're trying to say here. Maybe a little bit shorter but more meaningful.